composer.json

```diff
@@ -11,17 +11,17 @@
     ],
     "require": {
         "php": ">=8.1",
-        "helmich/typo3-typoscript-parser": "dev-master#0ccb3a6",
+        "helmich/typo3-typoscript-parser": "2.4.1 as dev-master",
```

Comment:
Temporary, until similar PR gets merged to Rector. Now there is circular reference preving to upgrade this package, this is how to make composer update pass.